src/app/[locale]/projects/detail/[id]/components/ProjectDetailTab.tsx

```diff
@@ -0,0 +1,269 @@
+                        externalIds: new Map<string, string>(Object.entries(data.externalIds)),
+                        additionalData: new Map<string, string>(Object.entries(data.additionalData)),
+                        externalUrls: new Map<string, string>(Object.entries(data.externalUrls)),
+                    
+                        // Roles
+                        group: data.businessUnit,
+                        projectResponsible: CommonUtils.isNullOrUndefined(projectResponsible)?{ name: "", email: "" }:{name: projectResponsible.fullName, email: projectResponsible.email},
+                        projectOwner: CommonUtils.isNullOrUndefined(projectOwner)?{ name: "", email: "" }:{name: projectOwner.fullName, email: projectOwner.email},
+                        ownerAccountingUnit: data.ownerAccountingUnit,
+                        ownerBillingGroup: data.ownerGroup,
+                        ownerCountry: data.ownerCountry,
+                        leadArchitect: {
+                            name: CommonUtils.isNullOrUndefined(data["_embedded"]["leadArchitect"])?"":data["_embedded"]["leadArchitect"]["fullName"],
+                            email: CommonUtils.isNullOrUndefined(data["_embedded"]["leadArchitect"])?"":data["_embedded"]["leadArchitect"]["email"],
+                        },
+                        moderators: CommonUtils.isNullOrUndefined(data["_embedded"]["sw360:moderators"])?[]:
+                            data["_embedded"]["sw360:moderators"].map((moderator: any) => ({name: moderator.fullName, email: moderator.email})),
+                        contributors: CommonUtils.isNullOrUndefined(data["_embedded"]["sw360:contributors"])?[]:
+                            data["_embedded"]["sw360:contributors"].map((contributor: any) => ({name: contributor.fullName, email: contributor.email})),
+                        securityResponsibles: responsibles.map((elem: any) => { return { name: elem.fullName, email: elem.email } }),
+                        additionalRoles: new Map<string, string>(Object.entries(data.roles)),
+
+                        // Project Vendor
+                        vendorFullName: CommonUtils.isNullEmptyOrUndefinedArray(data["_embedded"]["sw360:vendors"])?"":data["_embedded"]["sw360:vendors"][0]["fullName"],
+                        vendorShortName: CommonUtils.isNullEmptyOrUndefinedArray(data["_embedded"]["sw360:vendors"])?"":data["_embedded"]["sw360:vendors"][0]["shortName"],
+                        vendorUrl: CommonUtils.isNullEmptyOrUndefinedArray(data["_embedded"]["sw360:vendors"])?"":data["_embedded"]["sw360:vendors"][0]["url"]
+                    }
+                )
+            setLoading(false)
+            } catch(e) {
+                console.error(e)
+            }
+        })()
+
+        return () => controller.abort()
+    }, [])
+
+    return (
+        <>
+            <div className="ms-5 mt-2">
+                <Tab.Container defaultActiveKey="summary">
+                    <Row>
```

Comment:
I created a global component to define Sidebar menu you can refer to component detail page

Reply:
I feel that the React-Bootstrap Tab component is more generic and neat to use. I was wondering if we could we shift to that? What do you suggest?

Reply:
I agree with @deo002 
@hoangnt2 The global SideBar component was still not optimal and is not generic at all, making really har to portable usage.
I looked around to see how both bootstrap react Tab and NextUI Tab, and they are implemented in a neat separation way, so i do suggest we go with Bootstrap version for now